.vscode/settings.json

```diff
@@ -16,5 +16,6 @@
       "command": "direnv allow .",
       "runIn": "terminal"
     }
-  ]
+  ],
+  "GitHooks.hooksDirectory": "/Users/anthonymenefee/Documents/GitHub/helm-charts/.git/hooks"
```

Comment:
@anthonymenefee oops. I'm happy to add it to gitignore and as soon as it gets merged in we shouldn't have to deal with this anymore.

Reply:
hmm, if I change it up to .vscode/* it successfully ignores that file. So imma switch that up.